src/cata_tiles.cpp

```diff
@@ -518,6 +519,28 @@ void cata_tiles::load_tilejson_from_file(const std::string &tileset_dir, std::if
         load_tilejson_from_file(config, 0, newsize);
         offset = newsize;
     }
+
+    // allows a tileset to override the order of mutation images being applied to a character
+    if( config.has_array( "overlay_ordering" ) ) {
+        tileset_mutation_overlay_ordering.clear();
+        JsonArray orderarray = config.get_array( "overlay_ordering" );
+        while( orderarray.has_more() ) {
+            JsonObject ordering = orderarray.next_object();
+
+            int order = ordering.get_int( "order" );
+            if( ordering.has_array( "id" ) ) {
```

Comment:
You can use `get_tags`, which handles arrays of strings and a single string the same way:
```C++
for( auto &id : ordering.get_tags( "id" ) ) {
    tileset_mutation_overlay_ordering[id] = order;
}
```